server/api/categories.go

```diff
@@ -14,9 +14,11 @@ import (
 func (a *API) registerCategoriesRoutes(r *mux.Router) {
 	// Category APIs
 	r.HandleFunc("/teams/{teamID}/categories", a.sessionRequired(a.handleCreateCategory)).Methods(http.MethodPost)
+	r.HandleFunc("/teams/{teamID}/categories/reorder", a.sessionRequired(a.handleReorderCategories)).Methods(http.MethodPut)
```

Comment:
Probably I would ask @wiggin77 about this new endpoints routes to verify it matches with the approach that we are following for the new API endpoints. I think we can add new endpoints, but if we are consistent upfront, is going to be less painful later.

@wiggin77 can you confirm if this is inline with your new API design, and if not, how would you change the URLs there?

Reply:
This looks good to me.

Reply:
Awesome! Thanks!